packages/pxweb2/public/locales/no/translation.json

```diff
@@ -7,7 +7,8 @@
     "title": "Velkommen til PxWeb 2.0",
     "header": {
       "title": "PxWeb 2.0 [Main]",
-      "logo": "PxWeb 2.0"
+      "logo": "PxWeb 2.0",
+      "arialabelheader": "Hovenavigasjon"
```

Comment:
Hovenavigasjon -> Hovednavigasjon

Reply:
https://www.figma.com/design/E9hjvAml7Kd8svvXztA6LB/PxWeb-2.0-Ready-for-dev-and-prototypes?node-id=9639-72279&t=5P3Unryutsqexf5i-0 
I have changed Hovednavigasjon to "Velg språk" and other translated texts for the moment